tomviz/AboutDialog.cxx

```diff
@@ -20,27 +20,96 @@
 #include "ActiveObjects.h"
 #include "MainWindow.h"
 #include "ModuleManager.h"
+#include "PythonUtilities.h"
 #include "tomvizConfig.h"
 
 #include <pqApplicationCore.h>
 #include <pqSettings.h>
+
+#include <vtkNew.h>
 #include <vtkPVConfig.h>
+#include <vtkPVOpenGLInformation.h>
+#include <vtkVersion.h>
+
+#include <QString>
+#include <QTreeWidget>
+
+#include <itkConfigure.h>
 
 namespace tomviz {
 
+static inline void AddItem(QTreeWidget* tree, const QString& name,
+                           const QString& value)
+{
+  QTreeWidgetItem* item = new QTreeWidgetItem(tree);
+  item->setText(0, name);
+  item->setText(1, value);
+}
+
+static inline void AddPythonAttributeItem(QTreeWidget* tree,
+                                          const QString& itemName,
+                                          const char* module,
+                                          const char* attribute)
+{
+  Python::initialize();
+
+  bool itemAdded = false;
+  Python py;
+  Python::Module mod = py.import(QString(module));
+  if (mod.isValid()) {
+    Python::Function func = mod.findFunction(QString(attribute));
```

Comment:
Glad you didn't have to add anything to PythonUtilities, looks a little odd the naming thought, looking up an attribute with findFunction(...), I wonder if we need to do some renaming, but I can let it go :snowflake: